loki/transform/dependency_transform.py

```diff
@@ -300,7 +300,7 @@ def rename_imports(self, source, imports, targets=None):
         for im in imports:
             if im.c_import:
                 target_symbol = im.module.split('.')[0].lower()
-                if targets and target_symbol.lower() in targets:
+                if targets and target_symbol.lower() in targets and not 'func' in im.module.lower():
```

Comment:
Sorry for nitpicking further but just realized that this may fail in pretty trivial cases, e.g., `#include "myfunc.intfb.h"` will now not be processed... 😅 

Should we apply the IFS-specific knowledge a bit more explicitly, e.g. like this?
```py
target_symbol, *suffices = im.module.lower().split('.', maxsplit=1)
if targets and target_symbol in targets and 'func.h' not in suffices:
```
Note that this works even if no `.` is in the module name at all, returning `suffices` as an empty list.
That way this becomes a very loose restriction specific to the problem of statement functions and in-line with the coding standards.

Reply:
Ah of course, well spotted and apologies for the clumsy oversight 😅